collectors/python.d.plugin/sensors/metadata.yaml

```
@@ -1,54 +1,127 @@
+# yamllint disable rule:line-length
+---
 meta:
   plugin_name: python.d.plugin
   module_name: sensors
   monitored_instance:
     name: Linux Sensors (lm-sensors)
-    link: ''
+    link: https://hwmon.wiki.kernel.org/lm_sensors
     categories:
-    - data-collection.hardware-devices-and-sensors
-    icon_filename: 'microchip.svg'
+      - data-collection.hardware-devices-and-sensors
+    icon_filename: "microchip.svg"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - sensors
+    - temperature
+    - voltage
+    - current
+    - power
+    - fan
+    - energy
+    - humidity
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Examine Linux Sensors metrics with Netdata for insights into hardware health and performance. Enhance your system''s reliability with real-time hardware health insights.'
-    method_description: ''
+    metrics_description: >
```

here to preserve newlines you want vertical bar
```suggestion
    metrics_description: |
```